CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+Thanks for contributing.  A few tips:
+
+* When submitting new files, follow the folder structure listed under **Repository Setup** in [the README](./README.md).
+* Write each new sentence on a new line.
+GitHub allows leaving comments on a line-by-line basis.
+Review and comment on the content of submitted text is much easier if there are multiple lines on-which to leave comments.
+Sentences on consecutive lines will be collapsed into a single paragraph (like this one) in the  final rendring of the content.
+* When reviewing a pull request, make suggestions via regular comment.
+If you have a specific text to suggest, use the [the _GitHub_ "suggestion" feature][suggestion feature].
+
+
+[suggestion feature]: https://help.github.com/articles/commenting-on-a-pull-request/#adding-line-comments-to-a-pull-request
```

Comment:
```suggestion
[suggestion feature]: https://help.github.com/articles/commenting-on-a-pull-request/#adding-line-comments-to-a-pull-request and https://help.github.com/articles/incorporating-feedback-in-your-pull-request/
```

Reply:
This line is not visible in the rendered markdown.  It creates a shorthand way of representing the help article URL using the friendly text `suggestion feature` instead.  You can see me taking advantage of that in line 9.

Reply:
Included your link inline in the contributing text.

Reply:
Nice, that's a feature I didn't know yet. 👍

Reply:
👍